### ISSUE TITLE
How to set env variables ?

### ISSUE THREAD

**Issue by kleysonr** [damavis/airflow-pentaho-plugin#36]
Hi.

I need to set a new KETTLE_HOME value for the DAG execution.

Is that possible ?

**Comment by piffall** [comment]
Yes, just follow [this instructions](https://github.com/damavis/airflow-pentaho-plugin?tab=readme-ov-file#airflow-connection) assuming that `"pentaho_home"` is the same as `KETTLE_HOME`.

**Comment by kleysonr** [comment]
@piffall 

I use a single kettle.properties inside each project to set the variables env dependent.

So when each dag runs, for each project, the pdi should read the kettle.properties inside the project folder.

I can't see how to use the pentaho_home as a solution per each individual kettle.properties files.

Can you provide an example of configuration for the use-case ?

Or, is it possible add a new extra parameter for kettle_home ?

Best Regards.
Kleyson Rios.

**Comment by piffall** [comment]
Hi @kleysonr . In the case that you have kettle deployed multiple times and assuming that each of this paths contains a full deployment of PDI, you can just set multiple `connections`, each having `pentaho_home` pointing to corresponding path.

```
/opt/kettle_a
/opt/kettle_b
/opt/kettle_c
```
